server/@types/prisonRegisterImport/index.d.ts

```diff
@@ -0,0 +1,320 @@
+      200: {
+        content: {
+          'text/plain': unknown
+        }
+      }
+      /** Client error - invalid prisonId or similar */
+      400: {
+        content: {
+          'text/plain': string
+        }
+      }
+      /** The prison does not have a Offender Management Unit email address */
+      404: {
+        content: {
+          'text/plain': string
+        }
+      }
+    }
+  }
+  putEmailAddressForOffenderManagementUnit: {
+    parameters: {
+      path: {
+        prisonId: string
+      }
+    }
+    responses: {
+      /** The email address was created */
+      201: unknown
+      /** The email address was updated */
+      204: never
+      /** Client error - invalid prisonId, email address, media type or similar */
+      400: unknown
+      /** No prison found for the supplied prison id */
+      404: unknown
+    }
+    requestBody: {
+      content: {
+        'text/plain': string
+      }
+    }
+  }
+  deleteEmailAddressForOffenderManagementUnit: {
+    parameters: {
+      path: {
+        prisonId: string
+      }
+    }
+    responses: {
+      /** The email address was removed */
+      204: never
+      /** Client error - invalid prisonId or similar */
+      400: unknown
+    }
+  }
+  /** Updates prison information, role required is MAINTAIN_REF_DATA */
+  updatePrison: {
+    parameters: {
+      path: {
+        prisonId: string
+      }
+    }
+    responses: {
+      /** Prison Information Updated */
+      200: {
+        content: {
+          'application/json': components['schemas']['PrisonDto']
+        }
+      }
+      /** Information request to update prison */
+      400: {
+        content: {
+          'application/json': components['schemas']['ErrorResponse']
+        }
+      }
+      /** Unauthorized to access this endpoint */
+      401: {
+        content: {
+          'application/json': components['schemas']['ErrorResponse']
+        }
+      }
+      /** Incorrect permissions to make prison update */
+      403: {
+        content: {
+          'application/json': components['schemas']['ErrorResponse']
+        }
+      }
+      /** Prison ID not found */
+      404: {
+        content: {
+          'application/json': components['schemas']['ErrorResponse']
+        }
+      }
+    }
+    requestBody: {
+      content: {
+        'application/json': components['schemas']['UpdatePrisonDto']
+      }
+    }
+  }
+  /** All prisons */
+  getPrisons: {
+    responses: {
+      /** Successful Operation */
+      200: {
+        content: {
+          'application/json': unknown
+        }
+      }
+    }
+  }
+  /** Information on a specific prison */
+  getPrisonFromId: {
+    parameters: {
+      path: {
+        prisonId: string
+      }
+    }
+    responses: {
+      /** Successful Operation */
+      200: {
+        content: {
+          'application/json': components['schemas']['PrisonDto']
+        }
+      }
+    }
+  }
+  getPrisonFromId_1: {
```

Comment:
Is this getPrisonFromId_1 used?

Reply:
Not currently. That file was generated when I ran `npx openapi-typescript` and I didn't make any changes to it

I think it appended `_1` because `prison-register` has two endpoints with the same path variable:
```
'/prisons/id/{prisonId}': {
  /** Information on a specific prison */
  get: operations['getPrisonFromId']
}
'/gp/prison/{prisonId}': {
  get: operations['getPrisonFromId_1']
}
```